cli/src/cli.rs

```diff
@@ -154,6 +168,14 @@ pub enum Commands {
     /// verifying incoming data, and applying to the repository and consensus & governance status.
     Update,
     /// Broadcast relevant data to the p2p network.
+    ///
+    /// This may contain
+    ///
+    /// 1. Newly created commits
+    /// 2. Consensus progress
+    /// 3. Governance votes
+    /// 5. Known peers
+    /// 6. 1-5 of the above, sent from other peers.
```

Comment:
How about fixing it to "All of the above"?

Reply:
fixed